bokehjs/src/coffee/models/widgets/data_table.coffee

```diff
@@ -99,6 +99,11 @@ export class DataTableView extends WidgetView
     @grid.invalidate()
     @grid.render()
 
+    # XXX: Workaround for `@model.source.trigger('change')` not triggering an event within python.
+    # But we still need it to trigger render updates
+    @model.source.data = @model.source.data
```

Comment:
@Signapur just to clarify, do things not work without this self-assignment? I would expect just the `@model.source.change.emit()` below to be sufficient